showcase/src/app/demo-ui/pages/progress-indicator/demo-progress-indicator-page.component.html

```diff
@@ -0,0 +1,19 @@
+<h1 class="mat-display-3" translate>SHOWCASE.DEMO.PROGRESS_INDICATOR.TITLE</h1>
+<section class="stark-section">
+	<example-viewer
+		[extensions]="['HTML', 'SCSS', 'TS']"
+		filesPath="progress-indicator/progress-indicator"
+		[exampleTitle]="'SHOWCASE.DEMO.PROGRESS_INDICATOR.TITLE' | translate"
+	>
+		<button class="progress-indicator-button" color="primary" mat-raised-button (click)="showProgressIndicator()">
+			{{ show ? "Hide" : "Show" }}
+		</button>
+		<div [starkProgressIndicator]="progressIndicatorConfig">
+			<p class="progress-indicator-example">
```

Comment:
Can you remove this `<p>` and move the class to the `<span>`... just to make it simpler?